backend/src/main/resources/db/migration/V1_0_7__Add_Users_Personal_Information.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE credentials
+ADD COLUMN name VARCHAR(255);
+
+ALTER TABLE credentials
+ADD COLUMN surname VARCHAR(255);
+
+ALTER TABLE proposals
+DROP COLUMN published;
+
+ALTER TABLE proposals
+ADD COLUMN status_id INTEGER;
```

Comment:
If this column is mandatory we need to set a value (default or based on the previous "published" column) to all the existing proposals in the database or the data can get corrupted. It is not that important now since we can delete the DB schema, but it won't be possible in the future once we are live

Reply:
We have set the status by default to be REVIEW_PENDING.